docs/core/porting/windows-compat-pack.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Porting to .NET Core - Using the Windows Compatibility Pack
+description: Porting to .NET Core - Using the Windows Compatibility Pack
+keywords: .NET, .NET Core, Windows, Compatibility
+author: immol
+ms.author: mairaw
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ffa28aed-45dd-4b02-965f-768c467829c3
+---
+
+# Using the Windows Compatibility Pack
+
+Many customers have reported that the primary reason why they have trouble
+porting to .NET Core is that they depend on APIs and technologies that only
+exist in .NET Framework. The *Windows Compatibility Pack* is about providing a
+good chunk of these technologies so that building .NET Core applications as well
+as .NET Standard libraries becomes much more viable for existing code.
+
+This is a logical [extension of .NET Standard 2.0][API Expansion] which
+significantly increased the API set to ensure code largely already compiles
+as-is. But in order to keep .NET Standard understandable, this didn't include
```

Comment:
is understandable the best word here?

Reply:
Reworded

---
In reply to: [150916060](https://github.com/dotnet/docs/pull/3674#discussion_r150916060) [](ancestors = 150916060)